kuma/search/filters.py

```diff
@@ -192,3 +194,21 @@ def filter_queryset(self, request, queryset, view):
         queryset = queryset.extra(facets=facets)
 
         return queryset
+
+
+class HighlightFilterBackend(BaseFilterBackend):
+    """
+    A django-rest-framework filter backend that adds search term highlighting.
+
+    If the query string `highlight=0` is passed, we bypass highlighting.
+
+    """
+    def filter_queryset(self, request, queryset, view):
+        highlight = request.GET.get('highlight', True)
```

Comment:
This should be `request.QUERY_PARAMS.get('highlight') or True`
- request.GET is not the DRF way of things
- the 'highlight' query parameter could be an empty string, e.g. `?highlight=&q=yeah+yeah`